experiments/compositions/composition/api/v1/composition_types.go

```diff
@@ -153,9 +154,17 @@ func (s *Composition) Validate() bool {
 	s.Status.ClearCondition(ValidationFailed)
 	// Validate Expanders
 	message := ""
-	for i, expander := range s.Spec.Expanders {
+	for expanderIndex, expander := range s.Spec.Expanders {
 		if expander.Name == "" {
-			message += fmt.Sprintf("Expander: %d missing Name", i)
+			message += fmt.Sprintf("Expander: %d missing Name", expanderIndex)
```

Comment:
Nit: something like `spec.expanders[%d] missing name` might be clearer.  We could even use the validation errors from apimachinery, but it'll end up with something like  `spec.expanders[%d]`  IIRC